### PR TITLE
add bubble theme

### PR DESCRIPTION
**Pull request by embe221ed**
as mentioned in #344. Sharing the bubble theme

### CODE REVIEW

File: themes/bubble_theme.sh
Comment (erikw) [inline themes/bubble_theme.sh:1]: Please remove the `_theme` from the filme name, it's redundant :)

**Review by erikw — CHANGES_REQUESTED** [review]
The theme looks great, worked straight away. Just one small fix needed :)

File: themes/bubble_theme.sh
Comment (embe221ed) [reply themes/bubble_theme.sh:1]: tru :D

**Review by erikw — APPROVED** [review]
Great!

**Pull request merged**